scripts/azure-pipelines/install-environment.ps1

```diff
@@ -0,0 +1,39 @@
+# Ensures we have downloaded and extracted a build environment
+# into our Azure Pipelines VM before we attempt to build. If the
+# environment was already cached and hence already present
+# this script will detect that.
```

Comment:
Is that our own caching? Or the not available yet caching the commit description mentions?

Reply:
The not available yet caching the commit description mentions. The scripts are adapted from the AppVeyor's ones.

Reply:
I don't see why we would mention caching, or use some pseudo caching though when it's just a wild guess if Azure will implement it in the same way, in the same path.

Reply:
Let's change the sentence to "If the environment archive is already present, this script will just extract it."?